src/main/scala/scoverage/ScoverageKeys.scala

```diff
@@ -11,7 +11,12 @@ object ScoverageKeys {
   lazy val coverageAggregate = taskKey[Unit]("aggregate reports from subprojects")
   lazy val coverageExcludedPackages = settingKey[String]("regex for excluded packages")
   lazy val coverageExcludedFiles = settingKey[String]("regex for excluded file paths")
-  lazy val coverageMinimum = settingKey[Double]("scoverage-minimum-coverage")
+  lazy val coverageMinimum = settingKey[Double]("scoverage-minimum-coverage-stmt-total")
+  lazy val coverageMinimumBranchTotal = settingKey[Double]("scoverage-minimum-coverage-branch-total")
+  lazy val coverageMinimumStmtPerPackage = settingKey[Double]("scoverage-minimum-coverage-stmt-per-package")
+  lazy val coverageMinimumBranchPerPackage = settingKey[Double]("scoverage-minimum-coverage-branch-per-package")
+  lazy val coverageMinimumStmtPerFile = settingKey[Double]("scoverage-minimum-coverage-stmt-per-file")
+  lazy val coverageMinimumBranchPerFile = settingKey[Double]("scoverage-minimum-coverage-branch-per-file")
```

Comment:
Also I assumed you were just following suit here  with the `scoverage-minimum-coverage` and the `-`s, but these can just be normal phrases as the descriptions without the `-` separated words.

Reply:
i didn't know that :) will change